package.json

```diff
@@ -42,6 +42,9 @@
   "bugs": {},
   "homepage": "https://github.com/strimzi/strimzi-ui#readme",
   "dependencies": {
+    "@babel/core": "^7.10.5",
```

Comment:
With recent changes, babel should have been superceeded with TS. Where is this needed/used?

Reply:
I think this was a rebasing thing. Have rebased again & it's no longer there.